source/_posts/aint-no-bugs-on-me.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: Ain't No Bugs on Me
+date: 2017-05-02
+category: Notes
+lede: Learn how to monitor security feeds to stay ahead of the ongoing threats to your infrastructure.
+author: Robert C Jensen
+published: true
+---
+
+> There ain't no bugs on me
+> There ain't no bugs on me
+> There may be bugs on some of you mugs
+> But there ain't no bugs on me
+
+*Traditional folk song*
+
+### Background
+
+How do you become aware of security bugs in your software systems? Is it when
+a machine prompts you to install updates? Is it when an engineer bumps the
```

Comment:
Maybe you could add "Is it when an engineer... you could say 'when you read about it on Hacker News'...